src/main/java/org/folio/rest/helper/ExchangeRateHelper.java

```diff
@@ -32,4 +32,8 @@ public ExchangeRate getExchangeRate(String from, String to) {
       throw new HttpException(400, e.getMessage());
     }
   }
+
+  public Double calculateExchange(Double rate, Double amount) {
+    return amount * rate;
```

Comment:
We need to use Money API here and apply rounding, see PR how to apply rounding https://github.com/folio-org/mod-orders/pull/825

Reply:
Thanks, Used Money Api including rounding